auth0/src/main/java/com/auth0/android/provider/BrowserPicker.java

```diff
@@ -110,14 +110,13 @@ public BrowserPicker build() {
     @Nullable
     String getBestBrowserPackage(@NonNull PackageManager pm) {
         Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse("http://www.example.com"));
-        ResolveInfo webHandler = pm.resolveActivity(browserIntent,
-                Build.VERSION.SDK_INT >= Build.VERSION_CODES.M ? PackageManager.MATCH_ALL : PackageManager.MATCH_DEFAULT_ONLY);
+        ResolveInfo webHandler = pm.resolveActivity(browserIntent, PackageManager.MATCH_DEFAULT_ONLY);
```

Comment:
Fix: this line's purpose is to find the "default browser app" in the device. "Matching all" would return all the browsers and pick one from there, which is not always the default app.